src/OrbitGl/TimelineTicks.cpp

```diff
@@ -8,38 +8,39 @@
 namespace orbit_gl {
 
 static std::set<uint64_t> GenerateScales() {
-  std::set<uint64_t> scales;
+  // 10^x for each unit and middle points.
```

Comment:
Nit: I don't get the middle points part of the comment. Aren't the middle points NOT added to this set?

Reply:
This comment was intended to be part of the entire function. Shall I put it outside the body?

Reply:
Ah makes sense. Yes in that case please above the function!